src/shared/components/common/markdown-textarea.tsx

```diff
@@ -331,6 +337,24 @@ export class MarkdownTextArea extends Component<
     );
   }
 
+  getCharacterCounter(contentLength: number, maxLength: number) {
+    return (
+      <>
+        <div className="flex-grow-1"></div>
+        <div
+          className={classNames(
+            "px-2 py-1",
+            contentLength > maxLength ? "text-danger" : "text-muted"
+          )}
+        >
+          {contentLength}
```

Comment:
Might be nice to `toLocaleString` these numbers so that, for instance, comma thousands separators show up in US English.